charts/cni-metrics-helper/README.md

```diff
@@ -12,10 +12,22 @@ This chart provides a Kubernetes deployment for the Amazon VPC CNI Metrics Helpe
 First add the EKS repository to Helm:
 
 ```shell
-helm repo add eks https://aws.github.io/eks-charts
+$ helm repo add eks https://aws.github.io/eks-charts
 ```
 
-To install the chart with the release name `cni-metrics-helper` and default configuration:
+Ensure helm repository up to date
+
+```shell
+$ helm repo update eks
+```
+
+To identify the version you are going to apply
+
+```shell
+$ helm search repo eks/cni-metrics-helper --versions
+```
+
+To install the latest chart with the release name `cni-metrics-helper` and default configuration:
```

Comment:
Address the sample command is for `latest` version (depends on the last update time of `helm repo update`).